src/calculateRentalCost.js

```diff
@@ -28,7 +28,27 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const perDay = 40;
+  const firstOff = 20;
+  const secondOff = 50;
+  const firstLine = 3;
+  const secondLine = 7;
+
+  if (days === 0) {
+    return 0;
+  }
+
+  if (days < firstLine) {
+    return days * perDay;
+  }
+
+  if (days >= firstLine && days < secondLine) {
```

Comment:
```suggestion
  if (days < secondLine) {
```
Эту часть проверяет предыдущий if, если `days < firstLine`, случится return в предыдущем `if` и этот код не выполнится